CHANGELOG.rst

```diff
@@ -11,6 +11,11 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+Unreleased
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Internal refactoring of the sphinx utilities and the report generation code
```

Comment:
Besides the questions of whether or not we are introducing any breaking changes, I'd like to release this if you are ok so that we can ensure all is well when this is actually upgraded everywhere and we can close out these changes, without having to chance revisiting in the future if there were to be an issue.
Thoughts?

Reply:
Sure thing. I'll bump the version number.